### PR TITLE
Add the possibility of reading with more dataitens

### PR DESCRIPTION
**Pull request by rtorresmatheus**
Add the possibility of reading with more dataitens:

- Added full-time flow to all tags;
- Modified the type of variable responsible for sensor id. Because with uin8_t we can only have 255 sensors. We were having a problem that truncating the value.

### CODE REVIEW

File: src/parser.c [hunk 1]
```diff
@@ -719,10 +704,11 @@ struct l_queue *parser_config_to_list(const char *json_str)
 			break;
 		}
 
-		if (get_sensor_id(&config->sensor_id, jobjentry) < 0) {
+		if (get_sensor_id(&sensor_id_aux, jobjentry) < 0) {
```
Comment (luisfelipemisil) [inline src/parser.c]: I think the number zero should be replaced by a variable with an explicit name

File: src/parser.c [hunk 2]
```diff
@@ -719,10 +704,11 @@ struct l_queue *parser_config_to_list(const char *json_str)
...
 			err = true;
 			break;
 		}
+		config->sensor_id = sensor_id_aux;
 
 		if (get_schema(&config->schema, jobjentry) < 0) {
```
Comment (luisfelipemisil) [inline src/parser.c]: I think the number zero should be replaced by a variable with an explicit name.

**Review by luisfelipemisil — COMMENTED** [review]
Really nice code, immaculate as always, with only a few numbers I don't understand.

File: src/parser.c [hunk 3]
```diff
@@ -662,8 +645,9 @@ char *parser_data_create_object(const char *device_id, uint8_t sensor_id,
 	if (ptm == NULL) {
 		puts("The localtime() function failed");
 		has_err = true;
+	} else {
+		strftime(buf, 32, "%Y-%m-%dT%X.0%z", ptm);
```
Comment (AntonioJanael) [inline src/parser.c]: What does this 32 mean? Would it be worth assigning this value to a constant?

File: src/knot_cloud.c [hunk 4]
```diff
@@ -549,7 +549,7 @@ int knot_cloud_list_devices(void)
  *
  * Returns: 0 if successful and a KNoT error otherwise.
  */
-int knot_cloud_publish_data(const char *id, uint8_t sensor_id,
```
Comment (AntonioJanael) [inline src/knot_cloud.c:552]: Why change from uint8_t to int? What do you think about including a comment explaining the reason for using int?

**Review by AntonioJanael — CHANGES_REQUESTED** [review]

**Review by AntonioJanael — APPROVED** [review]

**Pull request merged**